users/users.go

```diff
@@ -64,13 +64,10 @@ type UserRepository interface {
 	RetrieveByID(ctx context.Context, id string) (User, error)
 
 	// RetrieveAll retrieves all users
-	RetrieveAll(ctx context.Context, offset, limit uint64, email string, m Metadata) (UserPage, error)
+	RetrieveAll(ctx context.Context, offset, limit uint64, ids []string, email string, m Metadata) (UserPage, error)
```

Comment:
Can you explain in the comment for what `ids` are used and eventually improve the arg name, please?

Reply:
So it's userID? not groupID?